### PR TITLE
Add OMG History chapter

### PR DESCRIPTION
**Pull request by bact**
- Using text and 4 figures from Bob's Word doc 2024-07-04
- The annex number is removed - to match styles in #1044

(Replacing #1008 - with approvals there)

This PR will only add the content but not include it to the navigation bar. That will be handle by PR #1055

### CODE REVIEW

File: docs/omg-history.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+2.3 to SPDX 3.0, as shown by looking at Figure 16’s left-side (SPDX 2.3 model)
+compared to its right-side (SPDX 3.0), shifting the SPDX name from Software
+Package Data Exchange to System Package Data Exchange and the scope of items it
+can convey in a bill of materials.
+
+![Figure 16 – SPDX 2.3 Model compared to the SPDX 3.0 Model](images/spdx23-spdx30-comparison.png "Figure 16 – SPDX 2.3 Model compared to the SPDX 3.0 Model")
+**Figure 16 – SPDX 2.3 Model compared to the SPDX 3.0 Model**
+
+The SPDX 3.0.1 model is available at:
+[https://github.com/spdx/spdx-3-model](https://github.com/spdx/spdx-3-model)
```
Comment (kestewart) [inline docs/omg-history.md:137]: SHould this be 3.0.1?

**Comment by bact** [comment]
@bobmartin3000 This one is replacing #1008

File: docs/omg-history.md (hunk 1 above)
Comment (bact) [reply docs/omg-history.md:137]: I'm not sure. @goneall this one supposed to be a model file/RDF?

Comment (bact) [reply docs/omg-history.md:137]: Should it be https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json ?

(That URL has been used in "5.12 Standard data format requirements" section of Conformance - https://github.com/spdx/spdx-spec/pull/1046, but the section has recently been removed.)

Comment (bact) [reply docs/omg-history.md]: It is now read

```Markdown
The SPDX 3.0.1 model is available at:
[https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json](https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json)
```

Comment (goneall) [reply docs/omg-history.md]: I just checked and the schema is not being published there.

@zvr - do you happen to know where the schema is published?  On release, we manually copy it over to the spdx.dev website, but I don't know if there is a location where it is automatically generated.

Comment (bact) [reply docs/omg-history.md]: @goneall for v3.0 it is at https://spdx.github.io/spdx-spec/v3.0/model/schema.json

But the new spec-parser will publish v3.0.1 one at https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json

See @zvr comment here: https://github.com/spdx/spdx-spec/pull/1046#discussion_r1713022625

**Comment by bobmartin3000** [comment]
We can drop the last couple lines from this - the ones with the urls.  Meant to mention that today.

Bob

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Arthit Suriyawongkul ***@***.***>
Sent: Sunday, August 11, 2024 7:38:56 PM
To: spdx/spdx-spec ***@***.***>
Cc: Robert A Martin ***@***.***>; Mention ***@***.***>
Subject: [EXT] Re: [spdx/spdx-spec] Add OMG History chapter (PR #1053)

@bact commented on this pull request. In docs/omg-history. md: > +candidate 2 of SPDX 3. 0 in February of 2024. This release candidate will give +tool creators and those who maintain the support libraries for working with +SPDX time to start


@bact commented on this pull request.

________________________________

In docs/omg-history.md<https://github.com/spdx/spdx-spec/pull/1053#discussion_r1713080177>:

> +candidate 2 of SPDX 3.0 in February of 2024. This release candidate will give
+tool creators and those who maintain the support libraries for working with
+SPDX time to start revising their projects in advance of the final version of
+the specification. For those not following the inner workings, debates, and
+discussion of the combined 3T-SBOM and SPDX 3.0 working group for the last 3
+years there will be a dramatic change in the SPDX model as it goes from SPDX
+2.3 to SPDX 3.0, as shown by looking at Figure 16’s left-side (SPDX 2.3 model)
+compared to its right-side (SPDX 3.0), shifting the SPDX name from Software
+Package Data Exchange to System Package Data Exchange and the scope of items it
+can convey in a bill of materials.
+
+Figure 16 – SPDX 2.3 Model compared to the SPDX 3.0 Model
+**Figure 16 – SPDX 2.3 Model compared to the SPDX 3.0 Model**
+
+The SPDX 3.0.1 model is available at:
+[https://github.com/spdx/spdx-3-model](https://github.com/spdx/spdx-3-model)


@goneall<https://github.com/goneall> for v3.0 it is at https://spdx.github.io/spdx-spec/v3.0/model/schema.json

But the new spec-parser will publish v3.0.1 one at https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json

See @zvr<https://github.com/zvr> comment here: #1046 (comment)<https://github.com/spdx/spdx-spec/pull/1046#discussion_r1713022625>

File: docs/omg-history.md (hunk 1 above)
Comment (bact) [reply docs/omg-history.md]: I decided to remove that three lines.
The model URL will be in the Serialization chapter, per this PR #1059

**Comment by bact** [comment]
As @bobmartin3000 suggested, I removed that last three lines with URLs.

The schema and ontology URLs will be included in the Serialization chapter by this PR https://github.com/spdx/spdx-spec/pull/1059 - after the JSON-LD context content.

**Review by kestewart — APPROVED** [review]

**Review by goneall — APPROVED** [review]
LGTM

**Pull request merged**